### DISCUSSION TITLE
Welcome to Chitralekha-App Discussions!

### DISCUSSION THREAD

**Discussion by aravinth** [AI4Bharat/Chitralekha#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Suggest new features
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akhilsmittal** [comment]
Hi suppose certain words are continuously repeating in the translation, cant it happen that after adding to glossary and again translating give translation using those words. That will make work lot more easier.

**Comment by akhilsmittal** [comment]
There is also one other issue, on find and replace page, tabs are placed very low so when you find some word and try to replace it, TAB "REPLACE IN ALL PAGES" is no longer visible and so we cant press that
